### DISCUSSION TITLE
how to change install path

### DISCUSSION THREAD

**Discussion by 91Eric** [netdata/netdata#18827] [Q&A] [answered]
Hi, I'm new in netdata,when I use https://github.com/netdata/netdata-nightlies/releases/ get .run file and run it, but failed for /opt in my machine is read-only and I can't chang it. so I try using --target option to change install path, but it does't work, /opt is use as install path, but I read the file .run and add some log, I found targetdir is set correctly, so is any way to change install path?(I don't want to build a run file from source). Thanks!

**Comment by ilyam8** [comment, marked as answer]
Hi. No, it is not possible to change the path for static builds (.run).

**Reply by 91Eric** [reply]
Thanks for your apply

**Comment by 91Eric** [comment]
您好，您的来信我已收到.
